scripts/global.gd

```diff
@@ -2,10 +2,89 @@ extends Node
 
 
 var p1_name = "Player 1"
-var MUSHROOM_COLOR = Color(0.86,0.61,0.43,1.00)
+var MUSHROOM_COLOR : Color = Color(0.86,0.61,0.43,1.00)
 var p1_color = MUSHROOM_COLOR
 var p2_name = "Player 2"
-var TARO_COLOR = Color(1.00,0.56,1.00,1.00)
+var TARO_COLOR : Color = Color(1.00,0.56,1.00,1.00)
 var p2_color = TARO_COLOR
 var num_games_index = 0
 var num_games = 1
+var sounds_dir = "user://sounds"
+var tts_voice = null
+
+var _base_sound_keys : Array[String] = [
+	"point",
+	"serve",
+	"wins the game",
+	"wins the match",
+	"deuce",
+	"match score",
+	"game score",
+	"victory chime", # (three notes ending with a longer chord)
+	"grand victory chime", # (three series of three notes, ending with a longer greater chord)
+	"point award chime", #  (a bell "ding")
+	"point rescind chime", # (a low "vwoopt" sound)
+]
+
+func _ready() -> void:
+	var voices = DisplayServer.tts_get_voices_for_language("en")
+	if voices.size() > 0:
+		tts_voice = voices[0]
```

Comment:
i am choosing the first voice here, but there can be many. i wonder if we should make a selector? not sure how it will be on android. the 0th voice for me has a carribean accent